README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
Yes, so for new non-local js/css files, they should be downloaded as local files and so can benefit from the rake 'inline'. Developers need to know this and so need a doc about it